kmq.js

```diff
@@ -36,6 +36,25 @@ client.on("message", (message) => {
     }
 });
 
+client.on("voiceStateUpdate", (oldMember, newMember) => {
+    let oldUserChannel = oldMember.voiceChannel;
+    let newUserChannel = newMember.voiceChannel;
```

Comment:
Documentation says that this should be `.channel`, are you sure this works?

Reply:
Yup! I tried with `.channel` and it doesn't work.